tests/scripts/github-action-helper.sh

```diff
@@ -121,7 +121,7 @@ function build_rook() {
   tests/scripts/validate_modified_files.sh build
   docker images
   if [[ "$build_type" == "build" ]]; then
-    docker tag "$(docker images | awk '/build-/ {print $1}')" rook/ceph:master
+    docker tag "$(docker images | awk '/build-/ {print $1}')" rook/ceph:local-build
```

Comment:
Is this broken? I use this all the time and we shouldn't need to change this to avoid pulling an image on Docker. That's strange.

Reply:
The issue I'm trying to avoid is that the CI should never test against a dockerhub tag. The CI should only test against a local build. The only exception to that rule is where the upgrade pulls a previous release from dockerhub. So if we tag the local build with something that is not available on dockerhub we will know that we are testing the right thing. 

At the same time, I don't want to disrupt the developer flow. Perhaps we could tag both with `master` and `local-build`, and in the release branch we would only tag with `local-build`. I think that would be sufficient. wdyt?

Reply:
Per discussion in huddle, this doesn't affect the dev flow. During development, anyway the image needs to be tagged separately.